pinot-plugins/pinot-file-system/pinot-adls/pom.xml

```diff
@@ -94,6 +95,21 @@
         <artifactId>msal4j</artifactId>
         <version>1.13.5</version>
       </dependency>
+      <dependency>
+        <groupId>net.minidev</groupId>
+        <artifactId>json-smart</artifactId>
+        <version>${jsonsmart.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.wildfly.openssl</groupId>
+        <artifactId>wildfly-openssl</artifactId>
+        <version>${wildfly-openssl.version}</version>
```

Comment:
@xiangfu0 We have noticed that by overriding the version within this module, this creates a discrepancy in versions relative to any shaded jars (1.0.7.Final in the shaded jar vs 1.1.3.Final here). This discrepancy is not ideal  as it would be preferable to manage the dependencies centrally from the root pom's dependency management section as suggested by @gortiz This also opens up the door to folks that need to override dependency versions in their builds for compliance reasons.
Given the above, would it be possible to move the version settings to dependencyManagement in the root pom?

Reply:
https://github.com/apache/pinot/pull/12597